run-accessibility-check.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Script to run the Accessibility Check locally
+# Constants to add colour to text
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+# Build docker (can comment out if already built)
+docker-compose build
+printf "*** ${BLUE}Begin Accessibility Check${NC} ***\n"
+pa11y localhost --config ./pa11y.json --include-notices --include-warnings --reporter html > reports/accessibility_check.html

```

Comment:
This only checks our home page for errors - is there a way to check other pages? What about the screens that are navigated to with React and don't have a unique- are those testable?

Reply:
This tool only allows us to generate reports according the URLs. So we are not able to navigate inside React components and generate reports for each pages.

However, there is a tool called _**react-a11y**_ that could be very interesting to configure to check the accessibility of our React components.

Ref: https://github.com/reactjs/react-a11y

For now, I updated the accessibility script to get reports from all current URLs, meaning one for the home page, one for the prototype page, one for the status page and one for the home page of the eMIB Sample Test.

Updates are in commit #https://github.com/code-for-canada/project-thundercat/pull/75/commits/e43c0da6dec4b9bbe707ad7f1c866ecb89970948

Reply:
See also commit #https://github.com/code-for-canada/project-thundercat/pull/75/commits/99fc76d238f9235fd55d1dd965597964a947690f